django_pesapal/urls.py

```diff
@@ -1,9 +1,8 @@
 from django.conf.urls import patterns, url
-from django.contrib.auth.decorators import login_required
 
 from .views import TransactionCompletedView
 
 urlpatterns = patterns(
     '',
-    url(r'^transaction/completed/$', login_required(TransactionCompletedView.as_view()), name='transaction_completed'),
+    url(r'^transaction/completed/$', TransactionCompletedView.as_view(), name='transaction_completed'),
```

Comment:
To allow the scenario where a user can purchase an item and make payment to the site without creating an account for the website.